projects/optic/src/utils/spec-loaders.ts

```diff
@@ -136,15 +136,18 @@ async function specFromInputToResults(
 
 // filePathOrRef can be a path, or a gitref:path (delimited by `:`)
 export const getFileFromFsOrGit = async (
-  filePathOrRef: string,
-  config: OpticCliConfig
+  filePathOrRef: string | undefined,
+  config: OpticCliConfig,
+  strict: boolean
 ): Promise<ParseResult> => {
   const file = await specFromInputToResults(
     parseSpecVersion(filePathOrRef, defaultEmptySpec),
     config,
     process.cwd()
   ).then((results) => {
-    validateOpenApiV3Document(results.jsonLike, results.sourcemap);
+    validateOpenApiV3Document(results.jsonLike, results.sourcemap, {
+      strictOpenAPI: strict,
```

Comment:
Fix to strictly check OpenAPI on `to` files, and not to strictly check on `from` files.

The default is strict check, we want to _not_ strictly check API files on from files, since you can never edit a `from` file.